src/plugin/systems.rs

```diff
@@ -1463,7 +1453,16 @@ pub fn update_character_controls(
                 }
             }
 
-            if let Ok(mut transform) = transforms.get_mut(entity_to_move) {
+            // NOTE: Update the translation of the Rapier's rigid body without editing the Bevy's transform for
```

Comment:
Ok, good that you find it. But we must do that here, because otherwise move of character controller will be recorded as user's modification which will be break interpolation. I guess we should compute velocity this or is it should be always zero @sebcrozet?

Unfortunately author of Rapier is so often offline, and he did not reply for this issue since above month.